pkg/cli/admin/release/mirror.go

```diff
@@ -251,7 +262,20 @@ func (o *MirrorOptions) Complete(cmd *cobra.Command, f kcmdutil.Factory, args []
 		client := coreClient.ConfigMaps(configmap.NamespaceLabelConfigMap)
 		return client, nil
 	}
-	o.PrintImageContentInstructions = true
+	if o.PrintImageSourceInstructions == "" {
+		fmt.Fprintf(o.Out, "WARNING: ImageContentSources and ImageDigestMirrorSet in the default printed out mirror instructions are deprecated and will be removed in a future release. Use --print-mirror-instructions=idms to allow the printing of instructions for ImageDigestSources and ImageDigestMirrorSet.\n")
+		o.PrintImageSourceInstructions = instructionTypeICSP
+	} else {
+		instructionType := strings.ToLower(o.PrintImageSourceInstructions)
+		switch instructionType {
+		case instructionTypeICSP:
+			fmt.Fprintf(o.Out, "WARNING: --print-mirror-instructions=icsp is deprecated and will be removed in a future release. Use --print-mirror-instructions=idms instead.\n")
```

Comment:
Warnings should be printed to ErrOut instead Out as in https://github.com/openshift/oc/blob/b186341885cab69f82673153985fb721b9f9ade6/pkg/cli/admin/mustgather/mustgather.go#L146